### PR TITLE
Add blog post about free-threaded CPython ecosystem rollout

### PR DESCRIPTION
**Pull request by rgommers**
## Text styling

- [x] The blog is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags in order to do so (with only one level-one header).
- [x] All links describe where they link to (for example, check the [Quansight labs website](https://labs.quansight.org/)).
- [x] Any kind of styling that the author uses (for example, bold for emphasis) is consistent throughout the blog.

## Non-text contents

- [x] Blog post featured image is in PNG or JPEG format, **not** SVG.
- [x] All content is represented as text (for example, images need alt text and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing gifs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [x] If it were to be read as plain text, the blog still makes sense and no information is missing.

### CODE REVIEW

File: apps/labs/posts/free-threaded-python-rollout.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+Yesterday, [py-free-threading.github.io](https://py-free-threading.github.io/) launched!
+It's both a resource with documentation around adding support for free-threaded
+Python, and a status tracker for the rollout across open source projects in the
+Python ecosystem. We hope and expect both of these to be very useful, with the
+status tracker providing a one-stop-shop to check the support status of the
+dependencies of your project (e.g., "what was the first release of a package on
+PyPI to support free-threaded Python?" or "are there nightly wheels and where
+can I find them?") and get an overview of ecosystem-wide progress.
+
+![Tracker for package compatibility with free-threaded CPython](/posts/py_free_threading_tracker.png)
```
Comment (ngoldbaum) [inline apps/labs/posts/free-threaded-python-rollout.md:26]: Not sure if it matters for the real site, but the image doesn't render in the preview.

File: apps/labs/posts/free-threaded-python-rollout.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+is, and why you should care. In summary: it is a major change to CPython that
+allows running multiple threads in parallel within the same interpreter. It is
+becoming available as an experimental feature in CPython 3.13. A free-threaded
+interpreter can run with the global interpreter lock (GIL) disabled - a
+capability that is finally arriving as a result of the efforts that went into
+[PEP 703 - Making the Global Interpreter Lock Optional in CPython](https://peps.python.org/pep-0703/).
+
+Why? Performance. Multi-threaded performance. It makes it significantly easier
+to write code that efficiently runs in parallel and will utilize multiple CPU
+cores effectively. The core count in modern CPUs continue to grow, while clock
```
Comment (ngoldbaum) [inline apps/labs/posts/free-threaded-python-rollout.md:47]:
```suggestion
cores effectively. The core counts in modern CPUs continue to grow, while clock
```

File: apps/labs/posts/free-threaded-python-rollout.md [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+supported](https://github.com/python/cpython/issues/121403).
+
+It's encouraging though that issues like the ones above do get understood and
+resolved fairly quickly. With a good test strategy, and over time also test
+suites of libraries that cover Python-level threading better (such tests are
+largely non-existent now in most packages), detecting or guarding against
+thread-safety issues does seem doable. That test strategy will have to be
+multi-pronged: from writing new tests and running tests in loops with `pytest-repeat` &
+co., to getting [ThreadSanitizer](https://clang.llvm.org/docs/ThreadSanitizer.html)
+to work in CI and doing integration-level and real-world testing with users.
```
Comment (ngoldbaum) [inline apps/labs/posts/free-threaded-python-rollout.md:115]: I really really like this section!

File: apps/labs/posts/free-threaded-python-rollout.md [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+complex packages like PyO3 (of key importance for projects using Rust) and
+PyTorch, to the sheer volume of smaller packages with extension modules.
+The lessons we are learning, as far as they are reusable, are going into the
+documentation at
+[py-free-threading.github.io](https://py-free-threading.github.io).
+
+Furthermore, we'd like to spend time on whatever may be impactful in helping
+the ecosystem adopt free-threaded CPython, from answering questions
+to helping with debugging - please don't hesitate to reach out or ping us on
+GitHub!
```
Comment (ngoldbaum) [inline apps/labs/posts/free-threaded-python-rollout.md:167]: Maybe add a note that this repo is for ecosystem-wide coordination, so try to open project-specific issues in the project issue tracker.

**Review by ngoldbaum — COMMENTED** [review]
This is a great post! Just a couple small comments.

File: apps/labs/posts/free-threaded-python-rollout.md (hunk 1 above)
Comment (rgommers) [reply apps/labs/posts/free-threaded-python-rollout.md:26]: Yeah I'll need to fix that - the rendering does work, so I did something wrong here - I'll figure it out

File: apps/labs/posts/free-threaded-python-rollout.md (hunk 4 above)
Comment (rgommers) [reply apps/labs/posts/free-threaded-python-rollout.md:167]: Ah good point - I didn't actually mention https://github.com/Quansight-Labs/free-threaded-compatibility/. Will extend this a bit.

**Comment by rgommers** [comment]
Thanks for the review @ngoldbaum! In it goes.

**Pull request merged**